src/ray/util/process.h

```diff
@@ -103,6 +103,8 @@ class Process {
   /// Waits for process to terminate. Not supported for unowned processes.
   /// \return The process's exit code. Returns 0 for a dummy process, -1 for a null one.
   int Wait() const;
+  std::string ExitReason() const;
+  static void SetExitStatus(pid_t pid, int status);
```

Comment:
Add a comment that status is the returned status code when the process terminates?